common/wireaddr.c

```diff
@@ -298,6 +298,8 @@ bool wireaddr_from_hostname(struct wireaddr *addr, const char *hostname,
 	hints.ai_socktype = SOCK_STREAM;
 	hints.ai_protocol = 0;
 	hints.ai_flags = AI_ADDRCONFIG;
+	if (!dns_ok)
+		hints.ai_flags = AI_NUMERICHOST;
 	gai_err = getaddrinfo(hostname, tal_fmt(tmpctx, "%d", port),
```

Comment:
AI_NUMERICHOST razor sharp !!!
hope this work all time...

here we give away control !, can't we just for now stop the call and for now just return false; 
and wait for PR `gethostbbyname_tor`